.vscode/settings.json

```diff
@@ -1,20 +1,91 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "editor.detectIndentation": false,
-  "editor.insertSpaces": true,
   "editor.tabSize": 2,
-  "editor.trimAutoWhitespace": true,
+  "editor.insertSpaces": true,
```

Comment:
This seems like cruft.

Reply:
i copied this whole file from our main repo. `insertSpaces` tells the editor to never use tabs.

Reply:
I was referring to all changes in this file. Doesn't seem necessary. For example, `"**/generated-docs": true,` doesn't even exist in this repo.

Reply:
probably a good idea to leave them in anyway, in case people copy this repo as a starter for a real project.

Reply:
Ok. One could argue that if a user was starting a real project they may want to start with their *own* set of rules. (I personally dislike hiding `node_modules` in the files tree as it quickly lets me know if I've done `npm i` or not)